app/routes/companyInterest/components/CompanyInterestPage.js

```diff
@@ -30,8 +30,7 @@ export const EVENT_TYPES = {
 export const OTHER_TYPES = {
   readme: 'Annonsere i readme',
   collaboration: 'Samarbeid med andre linjeforeninger',
-  itdagene: 'Stand på itDAGENE',
-  labamba_sponsor: 'Sponsing av LaBamba (Studentkjeller)'
+  itdagene: 'Stand på itDAGENE'
```

Comment:
I too did not see this in the email

Reply:
Got some inside info from bedkom wanting to remove this as well